### DISCUSSION TITLE
Clear commands

### DISCUSSION THREAD

**Discussion by hehe-ecksdee** [jagrosh/MusicBot#1293] [Questions]
Hi is there any way to clear commands and text generated by the bot? It gets quite spammy and so far i can only manually remove them. Thanks!

**Comment by MWE001** [comment]
I use InterPunct Bot to do just that. I set a rule to purge a specific channel of all or only specific messages starting with my command for example after X amount of seconds or minutes. it works really well for me. As well, I add rules for other channels as well.
